README.md

```diff
@@ -2,18 +2,155 @@
+
+There are multiple stateful operators available like `reduce`, `stateful_map` and `fold_window`. The complete list can be found in the [API documentation for all operators](https://docs.bytewax.io/apidocs/bytewax.dataflow). Below we use the `fold_window` operator with a tumbling window based on system time to gather events and calculate the number of times events have occurred on a per-user basis.
+
+```python
+import datetime
+from collections import defaultdict
+
+from bytewax.window import TumblingWindowConfig, SystemClockConfig
+
+cc = SystemClockConfig()
+wc = TumblingWindowConfig(length=datetime.timedelta(seconds=5))
+
+def build():
+    return defaultdict(lambda: 0)
+
+def count_events(results, event):
+    results[event["type"]] += 1
+    return results
+
+flow.fold_window("session_state_recovery", cc, wc, build, count_events)
+```
+
+Output mechanisms in Bytewax are managed in the [capture operator](https://docs.bytewax.io/apidocs/bytewax.dataflow#bytewax.dataflow.Dataflow.capture). There are a number of helpers that allow you to easily connect and write to other systems ([output docs](https://docs.bytewax.io/apidocs/bytewax.outputs)). If there isn’t a helper built, it is easy to build a custom version, which we will do below. Similar the input, Bytewax output can be parallelized and the client connection will occur on the worker.
+
+```python
+import json
+
+import psycopg2
+
+def output_builder(worker_index, worker_count):
+    # create the connection at the worker level
+    conn = psycopg2.connect("dbname=website user=bytewax")
+    conn.set_session(autocommit=True)
+    cur = conn.cursor()
+
+    def write_to_postgres(user_id__user_data):
+        user_id, user_data = user_id__user_data
+        query_string = '''
+                    INSERT INTO events (user_id, data)
+                    VALUES (%s, %s)
+                    ON CONFLICT (user_id)
+                    DO
+                        UPDATE SET data = %s;'''
+        cur.execute(query_string, (user_id, json.dumps(user_data), json.dumps(user_data)))
+    return write_to_postgres
+
+flow.capture(ManualOutputConfig(output_builder))
+```
+
+Bytewax dataflows can be executed on a single host with multiple Python processes, or on multiple hosts. Below is an example of running bytewax across multiple hosts. When processing data in a distributed fashion, Bytewax will ensure that all items with the same key are routed to the same host.
+
+```python
+if __name__ == "__main__":
+    addresses = [
+    "localhost:2101"
+    ]
+
+    cluster_main(
+        flow,
+        addresses=addresses,
+        proc_id=0,
+        worker_count_per_proc=2)
+```
+
+### Deploying and Scaling
+
+Bytewax can be run on a local machine, or remote machine just like a regular python script.
+
+```sh
+python my_dataflow.py
+```
+
+It can also be run in a docker container as described further in the [documentation](https://docs.bytewax.io/deployment/container).
```

Comment:
```suggestion
It can also be run in a Docker container as described further in the [documentation](https://docs.bytewax.io/deployment/container).
```